### PR TITLE
add "exposeSampleData" to webserviceproperties

### PR DESCRIPTION
**Pull request by NonStatic2014**

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@NonStatic2014__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (bohu). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

File: arm-machinelearning/2016-05-01-preview/swagger/webservices.json [hunk 1]
```diff
@@ -368,6 +368,10 @@
           "description": "The uri for the swagger spec associated with this web service.",
           "readOnly": true
         },
+        "exposeSampleData": {
+            "type": "boolean",
+            "description": "If true, the WebServiceProperties will expose sample data. Otherwise, it will not have it."
```
Comment (tudordid) [inline arm-machinelearning/2016-05-01-preview/swagger/webservices.json]: The description should say  "Flag that controls whether to expose sample data or not in the web service's swagger definition

File: arm-machinelearning/2016-05-01-preview/swagger/webservices.json [hunk 2]
```diff
@@ -370,7 +370,7 @@
         },
         "exposeSampleData": {
             "type": "boolean",
-            "description": "If true, the WebServiceProperties will expose sample data. Otherwise, it will not have it."
+            "description": "Flag this controls whether to expose sample data or not in the web service's swagger definition."
```
Comment (amarzavery) [inline arm-machinelearning/2016-05-01-preview/swagger/webservices.json]: Flag ~~this~~ **that** controls ...

File: arm-machinelearning/2016-05-01-preview/swagger/webservices.json [hunk 3]
```diff
@@ -368,6 +368,10 @@
           "description": "The uri for the swagger spec associated with this web service.",
           "readOnly": true
         },
+        "exposeSampleData": {
+            "type": "boolean",
+            "description": "Flag this controls whether to expose sample data or not in the web service's swagger definition."
```
Comment (amarzavery) [inline arm-machinelearning/2016-05-01-preview/swagger/webservices.json]: Flag ~~this~~ **that** controls ...

**Pull request merged**